.github/templates/main.yml

```
@@ -124,64 +153,73 @@ jobs:
       - build-wrappers-android
       - build-wrappers-linux
       - build-wrappers-windows-uwp
+    outputs:
+      nuget_pkg_version: ${{ steps.find-nupkg-version.outputs.nuget_pkg_version }}
     steps:
-      - name: Fetch Artifacts MacOS
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+      - #@ checkoutCode()
+      - name: Set version suffix
+        id: set-version-suffix
+        run: echo "::set-output name=build_suffix::${{ github.event_name == 'pull_request' && format('PR-{0}.{1}', github.event.number, github.run_number) || format('alpha.{0}', github.run_number)}}"
```

I wonder if this will work:

```suggestion
        #! Build suffix is PR-1234.5 for PR builds or alpha.123 for branch builds.
        run: echo "::set-output name=build_suffix::${{ github.event_name == 'pull_request' && format('PR-{0}', github.event.number) || 'alpha'}}.${{ github.run_number }}"
```